libs/coin-modules/coin-bitcoin/src/wallet-btc/wallet.ts

```diff
@@ -218,7 +218,7 @@ class BitcoinLikeWallet {
       onDeviceSignatureGranted,
       onDeviceStreaming,
     } = params;
-    let hasTimestamp = params.hasTimestamp;
+    const hasTimestamp = params.hasTimestamp;
```

Comment:
in fact, 
if I don't make mistake, all the "hasTimestamp" and the "initialTimestamp" logic can be removed from coin-btc module.
they are only used in peercoin.
We can remove these parameters in the related methods

Reply:
we can do it in another ticket if you want

Reply:
I'll jump in and do it on this PR @hzheng-ledger